example_10/doc/userdoc.rst

```diff
@@ -91,6 +91,96 @@ The *RRBot* URDF files can be found in the ``description/urdf`` folder.
     [RRBotSystemWithGPIOHardware]: Got command 0.5 for GP output 0!
     [RRBotSystemWithGPIOHardware]: Got command 0.7 for GP output 1!
 
+7. Let's introspect the ros2_control hardware component. Calling
+
+  .. code-block:: shell
+
+    ros2 control list_hardware_components
+
+  should give you
+
+  .. code-block:: shell
+
+    Hardware Component 1
+        name: RRBot
+        type: system
+        plugin name: ros2_control_demo_example_10/RRBotSystemWithGPIOHardware
+        state: id=3 label=active
+        command interfaces
+                joint1/position [available] [claimed]
+                joint2/position [available] [claimed]
+                flange_analog_IOs/analog_output1 [available] [claimed]
+                flange_vacuum/vacuum [available] [claimed]
```

Comment:
One general comment, should we indicate that flange_vacuum is the gpio mock hardware?

Reply:
I pushed an update, what do you think now?

Reply:
looks good. thanks for considering.